base.Makefile

```diff
@@ -46,6 +46,7 @@ TEMPLATE_DIRS ?= $(PYTHON_SRCDIR)/templates
 PYTHON_PKG_TOOLS ?= pip pip-tools setuptools wheel
 ### Commands (from `PYTHON_BINDIR` via `PATH` environment variable)
 DJLINT ?= djlint
+DJLINT_TARGETS ?= find $(TEMPLATE_DIRS) -type f -name '*.jinja2' | xargs $(DJLINT)
```

Comment:
La variable mélange à la fois la commande `$DJLINT` et les arguments `$TEMPLATE_FILES`, je pense que c'est une mauvaise pratique.

Reply:
Que proposes-tu pour éviter la répétition ?

Reply:
The suggestion avoids repetition (only twice) **but** makes the code a lot less readable. There is an indirection now, a coupling and `DJLINT_TARGETS` variable does not convey the real meaning of its value.

Choose your poison but I am unsure whether the added value for the sake of 'deduplication' is really worth it. KISS would be rather preferable.

My 2 cents.

Reply:
> Que proposes-tu pour éviter la répétition ?

Mon point c'était qu'on ne devrait pas mélanger djlint et les chemins, jamais.

Julien dit qu'il ne faut même pas factoriser les chemins trouvés par le find dans une variable.

Étant donné que les variables sont évaluées quelle que soit la target appelée (par exemple `clean`), je vais me ranger du coté de Julien pour l'instant et propose d'appeler la ligne complète 2 fois : 

```
find $(TEMPLATE_DIRS) -type f -name '*.jinja2' | xargs $(DJLINT) --check --lint
```

C'est un compromis : lisibilité, performance, maintenance :wink: